starwars/src/components/StarwarsChar.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import './StarWars.css';
+
+const StarwarsChar = props => {
```

Comment:
Hey Lee, this component looks really well organized 🔥  One thing you may want to try is to destructure the different props you're using like `eye_color`, `gender` etc... before you return your `JSX`. By doing that you'll be able to access those same props more succinctly `{props.gender}` vs `{props.starwarsChar.gender}`